bots/naughty/rhel-7-4

```diff
@@ -0,0 +1 @@
+rhel-7
```

Comment:
It's ok for now, I thought you were moving the files over and then linking, but this is a fairly simple cleanup.